lib/govuk_app_config/configure.rb

```diff
@@ -15,6 +15,19 @@
     true
   }
 
+  config.excluded_exceptions = [
```

Comment:
If an application has its own `config.excluded_exceptions` set, will the two conflict in some way? Might it be worth doing `+=` here?

Reply:
No, this just overwrites the raven-sentry's defaults (intentionally). If excluded exceptions is modified by the app it will happen after this code is run, so won't overwrite it.